docs/en-US/component/segmented.md

```diff
@@ -81,9 +81,9 @@ segmented/custom-style
 
 ### Slots
 
-| Name    | Description     |
-| ------- | --------------- |
-| default | option renderer |
+| Name    | Description                                                          |
+| ------- | -------------------------------------------------------------------- |
+| default | Option renderer. The scope parameter is ^[object]`{ item: Option }`. |
```

Comment:
You can refer to [here](https://element-plus.org/en-US/component/upload.html#slots) and add an extra column `type`.